app/modules/role/role.py

```diff
@@ -234,14 +234,14 @@ async def on_ready(self):
     @commands.Cog.listener()
     async def on_raw_reaction_add(self, payload: discord.RawReactionActionEvent):
         role = self.get_assigned_role(payload.guild_id, payload.channel_id, payload.message_id, payload.emoji.id)
-        if role is not None:
+        if role:
```

Comment:
These should be reverted to "if role is not None:", unless these are returning some kind of boolean instead of an object.

Reply:
I feel like it would be better to be explicit now, in case they decide to change how it works later and have Role be a non-null object that overrides __bool__.